compiler/runtime/OMRCodeCache.cpp

```diff
@@ -682,7 +687,15 @@ OMR::CodeCache::syncTempTrampolines()
             {
             void *newPC = (void *) TR::Compiler->mtd.startPC(entry->_info._resolved._method);
             void *trampoline = entry->_info._resolved._currentTrampoline;
-            if (trampoline && entry->_info._resolved._currentStartPC != newPC)
+            bool forceCreateTrampoline = false;
+
+            if (reinterpret_cast<intptr_t>(newPC) <= 0)
```

Comment:
the existence of temp trampoline means _currentStartPC must not be NULL, such that this fix could work.  however, `<=0` test is not right ... you cannot assume codeCache is not in 0x[8-F]..... areas. Rather, `J9TR_MethodNotCompiledBit` is defined for the testing ... low tagging really it is. Not sure it has been used in OMR side though.

Reply:
I was just thinking that this check might not be right. I will look at what it should be.